src/ds5/ds5-factory.cpp

```diff
@@ -962,8 +965,9 @@ namespace librealsense
               ds5_motion(ctx, group),
               ds5_advanced_mode_base(ds5_device::_hw_monitor, get_depth_sensor()),
               firmware_logger_device(ctx, group, ds5_device::_hw_monitor,
-                get_firmware_logs_command(),
-                get_flash_logs_command())
+                    get_firmware_logs_command(),
+                    get_flash_logs_command()),
+              ds5_thermal_tracking(ds5_device::_thermal_monitor)
```

Comment:
Why the `ds5_device::`?

Reply:
Dependency injection for registering user-defined callbacks

Reply:
Not sure I follow -- would it not work without the `ds5_device::` qualification?